contracts/NFTStorefront.cdc

```diff
@@ -67,7 +67,13 @@ pub contract NFTStorefront {
     // ListingCompleted
     // The listing has been resolved. It has either been purchased, or removed and destroyed.
     //
-    pub event ListingCompleted(listingResourceID: UInt64, storefrontResourceID: UInt64, purchased: Bool)
+    pub event ListingCompleted(
+        listingResourceID: UInt64, 
+        storefrontResourceID: UInt64, 
+        purchased: Bool,
+        nftType: Type,
+        nftID: UInt64
```

Comment:
Small nitpick, but given that `purchased` is a flag and the other fields are identifiers, it might make a bit more sense at the end of the event. Or does that break the upgradability of this change?

Reply:
Apparently preserving the current order makes it less likely to be a breaking change for event consumers.

Reply:
That makes sense!